octo-user-service/src/main/java/com/revature/octo/user/controller/BoardController.java

```diff
@@ -42,15 +51,95 @@
 	public ResponseEntity<List<SystemUser>> getAllUsersOnBoard(@PathVariable String boardId, HttpServletRequest request) {
 		int boardNum = Integer.parseInt(boardId);
 		System.out.println("boardId: " + boardId);
+		
 		List<SystemUser> systemUserListOnBoard = userRepo.findByBoardUserJoins_boardId(boardNum);
 		System.out.println("users on baord: " + systemUserListOnBoard.size());
+		
 		List<SystemUser> systemUserList = (List<SystemUser>) userRepo.findAll();
+		
 		System.out.println("users NOT on board: " + systemUserListOnBoard.size());
+		
 		for (SystemUser su : systemUserListOnBoard) {
 			systemUserList.remove(su);
 		}
 		System.out.println("Final: " + systemUserList);
 		return new ResponseEntity<List<SystemUser>>(systemUserList, HttpStatus.OK);
 		
 	}
+	
+	@GetMapping(path="/getUsersOnBoard/{boardId}", produces=MediaType.APPLICATION_JSON_VALUE)
```

Comment:
You didn't need this method. It exists on SystemUserController. Also, the parseInt is a needless step. You can cast it as an int in the method signature.